<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWidget</class>
 <widget class="QWidget" name="FilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>132</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QGroupBox exposes the title as QString (even if internally its probably a QLabel) */
QGroupBox {
	font-weight: bold;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Filters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_all">
        <property name="text">
         <string>All Files</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_allImages">
        <property name="text">
         <string>Images (jpg, png...)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_png">
        <property name="text">
         <string>Images PNG</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_jpeg">
        <property name="text">
         <string>Images JPEG</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_gif">
        <property name="text">
         <string>Images GIF</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBox_allVideo">
        <property name="text">
         <string>Videos (mpeg, avi...)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Fast Filtering</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <item>
       <widget class="QComboBox" name="fastFilteringComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>*.pdf, *.cpp, *.java</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>*.mp3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>*.html, *.htm, *.rtf, *.doc, *.pdf</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fastFilteringOnlyCheckBox">
        <property name="text">
         <string>Disable other filters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
